### PR TITLE
call self.launch() from def generate_slab() in main.py 

### PR DESCRIPTION
**Pull request by sakim8048**
error message: 

>   File "/global/cfs/cdirs/m4126/shikim/pynta/pynta/main.py", line 134, in generate_slab
>     self.rapidfire()
> AttributeError: 'Pynta' object has no attribute 'rapidfire'

This is resolved by changing rapidfire to `self.launch()`

### CODE REVIEW

File: pynta/main.py [hunk 1]
```diff
@@ -475,40 +476,24 @@ def launch(self):
         else:
             launch_multiprocess(self.launchpad,self.fworker,"INFO","infinite",self.num_jobs,5)
 
-    def execute(self,generate_initial_ad_guesses=True,calculate_adsorbates=True,
```
Comment (mjohnson541) [inline pynta/main.py:478]: I don't think this was intended? did you have an issue when rebasing?

File: pynta/main.py [hunk 2]
```diff
@@ -130,7 +130,8 @@ def generate_slab(self):
                 run_kwargs={"fmax" : 0.01},out_path=os.path.join(self.path,"slab.xyz"),constraints=["freeze half slab"])
             wfslab = Workflow([fwslab], name=self.label+"_slab")
             self.launchpad.add_wf(wfslab)
-            self.rapidfire()
+            self.launch()
+#            self.rapidfire()
```
Comment (mjohnson541) [inline pynta/main.py:134]: Can you remove the commented out line?

File: nersc-instructions [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#### this is a general instruction to install and launch pynta on nersc machines ###
```
Comment (mjohnson541) [inline nersc-instructions:1]: Should this be in with the documentation? Probably on a new page?

**Review by mjohnson541 — CHANGES_REQUESTED** [review]

File: nersc-instructions (hunk 3 above)
Comment (sakim8048) [reply nersc-instructions:1]: I am still updating the detail and not completed yet. I just temporarily wrote the note for myself and people who asked me about installing pynta on nersc. should I create a new page?

**Pull request closed without merging**